app/controllers/api/v1/users_controller.rb

```diff
@@ -60,6 +56,6 @@ def user
   end
 
   def check_owner
-    render_error(4011) unless user.id == current_user&.id
+    render_error(4003) unless user.id == current_user&.id
```

Comment:
This should be a different error. 4003 is a 401 error and this case should be a 403 error.